test/unit/generator/content/schedule/part2/index.test.js

```diff
@@ -4,32 +4,122 @@ const getTopUpDescription = require('../../../../../../app/generator/content/sch
 jest.mock('../../../../../../app/generator/content/schedule/part2/get-reduction-description')
 const getReductionDescription = require('../../../../../../app/generator/content/schedule/part2/get-reduction-description')
 
+jest.mock('../../../../../../app/generator/content/schedule/part2/get-reduction-zero-description')
+const getReductionZeroDescription = require('../../../../../../app/generator/content/schedule/part2/get-reduction-zero-description')
+
 const part2 = require('../../../../../../app/generator/content/schedule/part2')
 
-const { topUpSchedule, reductionSchedule } = require('../../../../../mocks/mock-schedule')
+const { topUpSchedule, reductionSchedule, reductionZeroSchedule, negativeRemainingAmountSchedule } = require('../../../../../mocks/mock-schedule')
 
 describe('schedule part 2', () => {
   afterEach(() => {
     jest.clearAllMocks()
   })
```

Comment:
Whitespace missing

Reply:
No white space missing, please check latest file : 
https://github.com/DEFRA/ffc-pay-statement-generator/blob/194721-Reduction-Schedule-content-update-for-0-payment/test/unit/generator/content/schedule/part2/index.test.js